data/mods/gen1/formats-data.ts

```diff
@@ -414,7 +414,7 @@ export const FormatsData: {[k: string]: ModdedSpeciesFormatsData} = {
 	slowbro: {
 		randomBattleMoves: ["amnesia", "surf", "thunderwave"],
 		exclusiveMoves: ["blizzard", "psychic", "rest", "rest"],
-		tier: "OU",
+		tier: "UUBL",
```

Comment:
If I understand it correctly, Slowbro is banned from UU but usable in OU regardless of how it's classified here, so I'm not worried about adjudicating a dispute here since it's purely cosmetic and we can easily push something to change it.